cookbooks/travis_internal_base/files/default/create-users

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# vim:filetype=sh
+
+main() {
+  : "${GITHUB_USERS_ENV:=/etc/default/github-users}"
+  : "${DEFAULT_GROUPS:=sudo}"
+
+  if [[ -f "${GITHUB_USERS_ENV}" ]]; then
+    # shellcheck source=/dev/null
+    source "${GITHUB_USERS_ENV}"
+  fi
+
+  DEFAULT_GROUPS=(${DEFAULT_GROUPS})
+
+  for pair in ${GITHUB_USERS}; do
+    __create_user "${pair%%:*}" "${DEFAULT_GROUPS[@]}"
+    __stub_zshrc "${pair%%:*}"
+    __add_github_keys "${pair%%:*}" "${pair##*:}"
+  done
+}
+
+__create_user() {
+  local username="${1}"
+  shift
+  local groups=("${@}")
+
+  if getent passwd "${username}"; then
+    return
+  fi
+
+  useradd \
+    -G "${groups[@]}" \
+    -U \
+    -d "/home/${username}" \
+    -m \
+    -s /bin/zsh \
+    "${username}"
+}
+
+__stub_zshrc() {
+  local username="${1}"
+  local zshrc="/home/${username}/.zshrc"
+
+  if [[ -s "${zshrc}" ]]; then
+    return
+  fi
+
+  echo '# this space intentionally left blank' >"${zshrc}"
```

Comment:
This is me intentionally setting the user's `~/.zshrc` to a file with a comment in order to avoid the default zsh behavior of triggering a setup flow when `~/.zshrc` is missing.  It's the same behavior and comment that was previously being done by the `travis_users` cookbook, fwiw.